src/core/Flora/Import/Package.hs

```diff
@@ -398,9 +403,8 @@ extractPackageDataFromCabal userId repository@(repositoryName, repositoryPackage
   let releaseId = deterministicReleaseId packageId packageVersion
   timestamp <- Time.currentTime
   let sourceRepos = getRepoURL packageName packageDesc.sourceRepos
-  let rawCategoryField = packageDesc ^. #category % to Cabal.fromShortText % to Text.pack
-  let categoryList = fmap (Tuning.UserPackageCategory . Text.stripStart . Text.stripEnd) (Text.splitOn "," rawCategoryField)
-  categories <- liftIO $ Tuning.normalisedCategories <$> Tuning.normalise categoryList
+  let categoryNames = map extractNameFromFloraCategories floraCategories
```

Comment:
Similary, this diff is problematic because you went from "normalise the categories taken from the cabal file" to "take all the canonical categories and forsake the categories of the cabal file"